docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +296,46 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+There is a known issue that the computing power of modern hardware is growing faster than memory. Using more and more modern HW CPUs, we may never reach the theoretical peak performance of some models due to the memory bound problem. `Sparse weights decompression feature` allows to achieve greater performance on such models just due to more reasonable memory consumption.
+
+Sparse matrix is a matrix in which most of the elements are zero. To use memory more efficiently, we can store only the values of non-zero elements of such matrices in memory using special storage structures (e.g., CSC - Compressed Sparse Column or CСS - Compressed Column Storage). Using this, we can reduce the memory load and accordingly achieve higher performance on the models.
```

Comment:
We are not usinig CSC or CCS actually. So just remove the part in braces.

Reply:
removed, thanks